python-sdk/docs/CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+## 1.2.3
+
+### Bug fixes
+* Fix the dataframe operator behavior of appending to it's output table instead of replacing. When `if_exists` is set to `replace`, replace the table rather than append [#1260](https://github.com/astronomer/astro-sdk/issues/1260)
+* Previously, `PostgresHook` did not support `database` kwarg. After schema is renamed to database in airflow as per this [PR](https://github.com/apache/airflow/pull/26744), now the user needs to set database to the table's database. [#1276](https://github.com/astronomer/astro-sdk/pull/1276)
```

Comment:
```suggestion
* Support `database` kwarg in `PostgresHook`. After schema is renamed to database in airflow as per this [PR](https://github.com/apache/airflow/pull/26744), the user can now set database instead of schema. [#1276](https://github.com/astronomer/astro-sdk/pull/1276)
```

Reply:
https://astronomer.slack.com/archives/C03868KGF2Q/p1669131445686549?thread_ts=1669130287.404939&cid=C03868KGF2Q -- What about backwards compatibility with old postgres provider?

We need to think this through before releasing this change

Reply:
Created an issue to check backward compatibility for PostgresHook https://github.com/astronomer/astro-sdk/issues/1291

Reply:
This need to be included in 1.2.3 as per latest discussion.

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/1289/commits/2370dd1c2550667d3a2a180268076f59e800061b)